pyproject.toml

```diff
@@ -32,5 +32,9 @@ __version__ = "{version}"
 [tool.pytest.ini_options]
 asyncio_mode = "auto"
 
+[tool.isort]
```

Comment:
Please remove this section. ts_pre_commit_conf takes care of all pre-commit config files.